buildkite/bazelci.py

```diff
@@ -625,7 +641,7 @@ def has_flaky_tests(bep_file):
 
 def print_bazel_version_info(bazel_binary, platform):
     print_collapsed_group(":information_source: Bazel Info")
-    execute_command(
+    version_output = execute_command(
```

Comment:
I think because you capture the stdout of the process, it will no longer print it (but this method is called `print_bazel_version_info`). Maybe just `eprint(version_output)` after the call to `execute_command`?